dashboard/src/reducers/charts.ts

```diff
@@ -72,10 +67,6 @@ const chartsReducer = (state: IChartState = initialState, action: ChartsAction):
     case getType(actions.charts.selectReadme):
     case getType(actions.charts.errorReadme):
     case getType(actions.charts.errorChart):
-    case getType(actions.charts.selectValues):
-    case getType(actions.charts.selectSchema):
-      return { ...state, selected: chartsSelectedReducer(state.selected, action) };
-    case getType(actions.charts.selectSchema):
```

Comment:
Not sure if I'm misreading, but this looks like the `errorChart` type now falls through to the return below?

Reply:
yes, it's the same behavior than before (I just removed the cases for `selectSchema` and `selectValues`). The error is stored withing the `selected` object in the `chartsSelectedReducer` method.